src/utils/twitch.tsx

```diff
@@ -2,16 +2,25 @@ import PropTypes from "prop-types";
 import React, { ChildContextProvider } from "react";
 import { makeHOC } from "./hocs";
 
+interface TwitchExtBetaClientQueryParams extends TwitchExtClientQueryParams {
```

Comment:
Just a small naming suggestion

```suggestion
interface TwitchExtLegacyClientQueryParams extends TwitchExtClientQueryParams {
```